app/routes.test.js

```diff
@@ -10,7 +10,7 @@ import { routes } from './routes';
 import { baseUrl } from './config';
 import * as dashboardController from './pages/dashboard/controller';
 import * as taskListController from './pages/task-list/controller';
-import * as documentController from './documentController';
+import * as getDocumentByFileName from './helpers/api/dapi/getDocumentByFileName';
```

Comment:
can you just import and mock the function like we have done elsewhere?

```
import { getDocumentByFileName } from './helpers/api/dapi/getDocumentByFileName';

jest.mock('./helpers/api/dapi/getDocumentByFileName');
```
and then where it is used we shouldn't need to do
`getDocumentByFileName.getDocumentByFileName ...`